### PR TITLE
Include subdirectories in search for `.d.ts` files

### PR DESCRIPTION
**Pull request by andy-ms**
Fixes https://github.com/Microsoft/TypeScript/issues/9488
`output\react_router\lib` (among others) will now be generated.

### CODE REVIEW

File: typings/tsd.d.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@
 /// <reference path="source-map-support/source-map-support.d.ts" />
 /// <reference path="node-fetch/node-fetch.d.ts" />
 /// <reference path="rimraf/rimraf.d.ts" />
+/// <reference path="recursive-readdir/recursive-readdir.d.ts" />
```
Comment (mhegazy) [inline typings/tsd.d.ts]: we should just remove this file and use `@types`.

**Comment by mhegazy** [comment]
👍

**Comment by mhegazy** [comment]
Do you mind doing another publish after this is in.

File: typings/tsd.d.ts (hunk 1 above)
Comment (andy-ms) [inline typings/tsd.d.ts]: See #54

**Comment by dalgard** [comment]
Have the types been republished after this fix?

I'm still getting an error with `@types/react-router`.

**Comment by RyanCavanaugh** [comment]
Next republish is scheduled for when the merge from `master` to `types-2.0` finishes, which should be today or tomorrow. After that we'll be publishing on a more consistent rolling basis.

**Comment by dalgard** [comment]
Sounds good – thanks!

**Pull request merged**